docs/glossary.md

```diff
@@ -19,34 +19,62 @@ See [this definition](https://helm.sh/docs/topics/charts/) of a Helm chart.
 A cluster refers to:
 * a [Kubernetes cluster](https://kubernetes.io/docs/concepts/architecture/) managed by Fleet
 * a `Cluster` [resource](https://github.com/rancher/fleet/blob/main/pkg/apis/fleet.cattle.io/v1alpha1/cluster_types.go#L59) in Fleet's API, which Fleet uses to manage that Kubernetes cluster
+
 ## Continuous Delivery
+
+Continuous delivery refers to the idea of enabling software to be produced in shorter cycles, with smaller increments
```

Comment:
That discussions seems very philosophical to me. Fleet as a tool can be used in both. Sources are contradicting themselves.
And no wonder, according to Wikipedia:

> The original CD book written by Jez Humble and David Farley (2010) popularized the term; however, since its creation the definition has continued to advance and now has a more developed meaning.
> ...
> Academic literature differentiates between continuous delivery and continuous deployment according to deployment method; manual vs. automated.

Well, it depends on how you trigger Fleet I guess.